definitions/ext-snowflake_account/definition.yml

```diff
@@ -0,0 +1,17 @@
+domain: EXT
+type: SNOWFLAKE_ACCOUNT
+
+synthesis:
+  identifier: snowflakeAccount
+  name: snowflakeAccount
+
+  conditions:
+    # The attribute’s value must match the provided value
+    attribute: eventName
```

Comment:
First, this should be an array 
eg
```
-attribute: eventName
```

Second, is there actually an attribute on telemetry with a key `eventName` and value `SnowflakeAccount` ? 

Can we see some real telemetry?

Reply:
@cianBuckley I was advised by JF that was the way to add a condition for Event data, by referring to an attribute called eventName that will refer to the name of the eventType itself. Can I refer to the actual name of the event? For example if it was reporting like `FROM SnowflakeAccount` how would I refer to the event name? Otherwise I will need to add an attribute to the data to identify it for example `entitySynthesisType:snowflakeAccount`

Reply:
@cianBuckley this should be fixed with my latest commit `a995ddc`

![Screenshot 2020-12-22 at 17 11 18](https://user-images.githubusercontent.com/3629903/102914900-d5e12b00-4478-11eb-9ddd-3ebe0638da25.png)